app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments,
+           def update_posts_counter
+             author.update(posts_counter: author.posts.count)
+           end
```

Comment:
- [ ] It would be nice to use the `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private
```suggestion

after_save :update_post_counter

  private

           def update_posts_counter
             author.update(posts_counter: author.posts.count)
           end
```

Reply:
This change has been implemented well